tests/bofire/benchmarks/test_detergent.py

```diff
@@ -0,0 +1,18 @@
+from bofire.benchmarks.detergent import Detergent
+from bofire.data_models.strategies.random import (
+    RandomStrategy as RandomStrategyDataModel,
+)
+from bofire.strategies.api import RandomStrategy
+
+
+def test_detergent():
+    d = Detergent()
+    assert len(d.domain.inputs) == 5
+    assert len(d.domain.outputs) == 5
+    assert len(d.domain.constraints) == 2
+
+    # check that the domain has feasible points
+    random_search_dm = RandomStrategyDataModel(domain=d.domain)
+    random_search = RandomStrategy(random_search_dm)
+    candidates = random_search.ask(2)
```

Comment:
Can you also add a call to f?

```
d.f(candidates)
```

So that it is checked that it is functional?